MediaBrowser.Providers/Plugins/TheTvdb/TvdbSeasonImageProvider.cs

```diff
@@ -90,6 +90,8 @@ private IEnumerable<RemoteImageInfo> GetImages(Image[] images, string preferredL
         {
             var list = new List<RemoteImageInfo>();
             var languages = _tvdbClientManager.GetLanguagesAsync(CancellationToken.None).Result.Data;
+            // any languages with null ids are ignored
+            languages = Array.FindAll(languages, language => language.Id != null);
```

Comment:
```suggestion
            var languages = _tvdbClientManager.GetLanguagesAsync(CancellationToken.None).Result.Data.Where(x => x.Id.HasValue);
```
Since the languages array is only being used in the .FirstOrDefault call, filtering out the nulls may not even be worth it, we could just check for null in that predicate and skip this change. I'd favor using `Where` over `Array.FindAll` to at least avoid allocating a new array. 
I'd usually avoid multi-eval of IEnumerables, but FirstOrDefault usage over a Where filter shouldn't be any less efficient, since the data-walk should be the same (0 -> first match)

Reply:
made the recommended change